core/src/main/java/dk/alexandra/fresco/lib/compare/bool/ParBinaryGreaterThanProtocolImpl.java

```diff
@@ -104,18 +100,18 @@ public ProtocolProducer nextProtocolProducer() {
 		} else {
 			return null;
 		}
-		return blb.getProtocol();
+		return blb.getProtocol();*/ return null;
```

Comment:
null will break the computation - should this be removed?

Reply:
I am deleting this since we never refer to it, and it's stated in the doc that it was worse than the simpler implementation